tests/test_definitions/column_aggregate_expectations/expect_column_median_to_be_between.json

```diff
@@ -177,6 +177,9 @@
           "max_value": 0,
           "catch_exceptions": false
         },
+        "suppress_test_for": [
```

Comment:
I don't know how this json file is used. Why did we end up needing to suppress spark here?